Dra 1860 validate rights input

VictorHarbo opened the pull request:
Validate that all expected values are present in object send to calculate method

davidgrove73 reviewed (commented):
Forbidden API fails when running mvn package . Maybe the Util class needs to be added to classes excluded from the forbidden API check (in pom.xml).

Other than that i looks fine. We should probably use the null check on other inputDTO's :-)

VictorHarbo commented:
I've switched form using reflection directly as that's a bad practice according to forbidden API. I've implemented the logic using Introspector instead.
